tests/unit/App.spec.ts

```diff
@@ -16,11 +15,11 @@ import EntityInfo from '@/components/common/EntityInfo.vue'
 import Stepper from '@/components/common/Stepper.vue'
 import { ConfirmDialog } from '@bcrs-shared-components/confirm-dialog'
 import mockRouter from './MockRouter'
-import Vuelidate from 'vuelidate'
+import { vi } from 'vitest'
```

Comment:
Can we include these imports in a common `setup.ts` file like JEST? Asking because I have to do the same thing in `sbc-common-components`

Reply:
Yes! There is a `setup.ts` in the `tests/unit` directory in this PR. It might be a good idea to move this import there also.

Reply:
Maybe we can keep this out of scope of this PR and do it later? I'm hoping to get this PR merged asap.